extensions/positron-python/src/client/positron/manager.ts

```diff
@@ -116,7 +118,14 @@ export class PythonRuntimeManager implements IPythonRuntimeManager {
      */
     public registerLanguageRuntime(runtime: positron.LanguageRuntimeMetadata): void {
         const extraData = runtime.extraRuntimeData as PythonRuntimeExtraData;
+        const pythonVersion: PythonVersion = toSemverLikeVersion(parseVersion(runtime.languageVersion));
```

Comment:
this seems to be the best way to get from a version string to the `PythonVersion` type. `toSemverLikeVersion` is only used in legacy code upstream, so it has been deprecated. 

I'm happy to put together a custom function for us if we don't want the deprecated function though.